### DISCUSSION TITLE
[station_logbooks] Few questions on PR and Bugs

### DISCUSSION THREAD

**Discussion by T012m3n7oR** [magicbug/Cloudlog#1480] [Q&A] [answered]
Hello Everyone!

The last two weeks or so I've been setting up my production setup mirrored to a second server so that I could get a development branch up and running without breaking my production. I should be able to get back into the swing of things now and contribute again!

I was wondering if you wanted bug reports for your dev branch?
And if yes, do you consider non friendly errors bug reports?

<img width="912" alt="image" src="https://user-images.githubusercontent.com/48076977/168807511-28c99386-21b7-4979-a114-b2d08861b912.png">
[Logbook_Model.php Line 1004](https://github.com/magicbug/Cloudlog/blob/6fe44f3b579200b37df3763c87bc53de1c294a0a/application/models/Logbook_model.php#L1004)

This then is receiving data from the list_logbook_relationships() function in [Logbook**s**_Model.php Line 177](https://github.com/magicbug/Cloudlog/blob/6fe44f3b579200b37df3763c87bc53de1c294a0a/application/models/Logbooks_model.php#L177)

This function returns a bool if there are no records found, and then errors when opening logbook with a non-obvious error.  I believe this function should either return records or an empty array - if it returns an empty array the front end should add a user friendly error/warning that there are No station logbooks defined.

**Comment by magicbug** [comment, marked as answer]
Yes issues are accepted for the dev branch, the where_in is when a Station Logbooks been created but has no Station Locations so displaying a friendly message would be handy I mean this will only show for users coming from v1 but needs sorting
